src/dispatching.rs

```diff
@@ -104,6 +104,9 @@
 //!
 //! Finally, we define our endpoints like this:
```

Comment:
Does it make sense to expand this sentence, since you've hidden the definitions? Something along the lines of "Finally, we define our endpoints **via simple async functions** like this:".

Reply:
Fixed.